network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/ConnectablePositionImpl.java

```diff
@@ -90,28 +217,36 @@ public ConnectablePositionImpl(C connectable,
         this.positionAttributesGetter3 = positionAttributesGetter3;
     }
 
-    private FeederImpl getFeeder(Function<Connectable<C>, ConnectablePositionAttributes> positionAttributesGetter) {
+    private FeederImpl getFeeder(Function<Connectable<C>, ConnectablePositionAttributes> positionAttributesGetter, ThreeSides side) {
         return (positionAttributesGetter != null && positionAttributesGetter.apply(getExtendable()) != null) ?
-                new FeederImpl(positionAttributesGetter) : null;
+                new FeederImpl(connectable -> {
```

Comment:
Why not adding the side as a new argument to FeederImpl constructor ? 
See remark above

Because, it's strange to override the side field in ConnectablePositionImpl class, each time a getFeeder method is called ...